x-pack/qa/kerberos-tests/src/test/resources/plugin-security.policy

```diff
@@ -0,0 +1,5 @@
+grant {
+  permission javax.security.auth.AuthPermission "doAsPrivileged";
+  permission java.io.FilePermission "${test.userkt.keytab}", "read";
```

Comment:
we should be able to remove this permission if we add the keytab as a resource within gradle?